src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1012,10 +1012,23 @@ protected override void OnApplyTemplate()
 			}
 #endif
 
+#if UNO_HAS_MANAGED_SCROLL_PRESENTER
+			Presenter?.UnhookScrollEvents(this);
+#endif
 			if (scpTemplatePart is ScrollContentPresenter presenter)
 			{
 				presenter.ScrollOwner = this;
 				Presenter = presenter;
+
+				// Note: the way WinUI does scrolling is very different, and doesn't use
+				// PointerWheelChanged changes, etc.
+				// We can either subscribe on the ScrollViewer or the SCP directly, but due to
+				// the way hit-testing works (see #16201), the SCP will not receive any pointer
+				// events. On WinUI, this is also the case: pointer presses are received on the SV,
+				// not on the SCP.
```

Comment:
Even if the SV and SCP are weird controls, this is suspicious ... we are not receiving any pointer events on the SCP on WinUI? It is a real feature or a bug?

Reply:
It is very suspicious indeed. Yes, we do NOT receive any pointer events on the SCP on WinUI: we don't get WheelChanged at all, and we get Pressed and Released on the SV only. But I guess it makes some kind of sense? In theory, the SCP doesn't really draw anything, it's just a boring ContentPresenter that takes care of scrolling, so it it has nothing that hit-test can check against; ergo, it's not hit-testable and doesn't receive pointer events.